src/core/http/types.h

```diff
@@ -47,6 +47,7 @@ namespace Http
     const QString CONTENT_TYPE_GIF = "image/gif";
     const QString CONTENT_TYPE_HTML = "text/html; charset=UTF-8";
     const QString CONTENT_TYPE_JS = "text/javascript; charset=UTF-8";
+    const QString CONTENT_TYPE_JSON = "application/json; charset=UTF-8";
```

Comment:
There is no **charset** parameter for this content type. See [RFC 4627](http://www.rfc-editor.org/rfc/rfc4627.txt).